api/routes/cartRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const cartController = require('../controllers/cartController')
+const {loginRequired} = require('../utils/auth')
+const router = express.Router();
+
+router.post('/createcart',loginRequired,cartController.createCart)
+router.patch('/updateCart',loginRequired,cartController.updateCart)
+router.get('/getCartByUserId',loginRequired,cartController.getCartByUserId)
+router.delete('/deleteCartByCartId/:cartId',loginRequired,cartController.deleteCartByCartId)
```

Comment:
restful api에서 엔드포인트에 동사를 적지 않아도 될 것 같습니다~!
```suggestion
router.post('/',loginRequired,cartController.createCart)
router.patch('/',loginRequired,cartController.updateCart)
router.get('/',loginRequired,cartController.getCartByUserId)
router.delete('/:cartId',loginRequired,cartController.deleteCartByCartId)
```